src/NUnitFramework/framework/Attributes/ValuesAttribute.cs

```diff
@@ -96,7 +96,7 @@ public ValuesAttribute(params object[] args)
         /// <summary>
         /// Get the collection of _values to be used as arguments
         /// </summary>
-        public IEnumerable GetData(ParameterInfo parameter)
+        public IEnumerable GetData(Type fixtureType, ParameterInfo parameter)
```

Comment:
The argument is not even used since the values needed are already in the data collection. Yet the change to this signature requires all derived classes of the attribute to also take this argument, so as to pass it to the base class.

This change is in turn motivated by the change of the interface.

I'm seeing a ripple effect here, where the fact that we need ReflectedType in a very few places is making us change in many more places. Maybe we need ot investigate more deeply why and where we use ReflectedType and get it out of the code entirely.